api/client/src/main/java/org/projectnessie/client/auth/oauth2/RefreshTokensFlow.java

```diff
@@ -30,7 +30,7 @@ class RefreshTokensFlow extends AbstractFlow {
 
   @Override
   public Tokens fetchNewTokens(@Nullable Tokens currentTokens) {
-    Objects.requireNonNull(currentTokens);
+    Objects.requireNonNull(currentTokens, "currentTokens is null");
     Objects.requireNonNull(currentTokens.getRefreshToken());
```

Comment:
Why not this one?

Reply:
Which one?

Reply:
Not sure BTW why the arg is `@Nullable` but the impl requires non-null

Reply:
> Which one?

```java
Objects.requireNonNull(currentTokens.getRefreshToken(), "currentTokens.getRefreshToken() is null");
```

> Not sure BTW why the arg is @Nullable but the impl requires non-null

The interface is used both for initial token fetches and token refreshes; `currentTokens` is null for initial token fetches and non-null for token refreshes.